### ISSUE TITLE
Create events page or integrate Meetup feed

### ISSUE THREAD

**Issue by ryanlalchand** [codeyoungstown/next-codeyoungstown.com#6]
Not sure if this is something you would want, or if there is a plugin or something for Meetup feeds, just an idea

**Comment by JoeDuncko** [comment]
Yes! That'd be cool. Idk what options there are for that though. Would need to do research.
